src/Uno.UWP/Devices/Bluetooth/GenericAttributeProfile/GattReadResult.cs

```diff
@@ -1,10 +1,17 @@
-﻿
+﻿#nullable enable
+
 namespace Windows.Devices.Bluetooth.GenericAttributeProfile
 {
 	public partial class GattReadResult
 	{
 		public GattCommunicationStatus Status { get; internal set; }
 		public Storage.Streams.IBuffer Value { get; internal set; }
 		public byte? ProtocolError { get; internal set; }
+
+		private GattReadResult(Storage.Streams.IBuffer buffer)
+		{
+			// dummy for Error CS8618  Non-nullable property 'Value' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.
+			Value= buffer;
```

Comment:
If it can't be null, let's create an empty buffer. See:
https://github.com/unoplatform/uno/blob/56d3ba98ee6cab055dc8d7c4941ce4494e20e946/src/Uno.UWP/System.Runtime/WindowsRuntimeBufferExtensions.cs#L10

Reply:
This constructor probably would be used in 'real code'.
This object cannot be created as such (no public ctor), it can only be returned inside other structs/classes from Bluetooth implementation.